src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9444,10 +9447,7 @@ internal ArgumentListSyntax ParseParenthesizedArgumentList()
                 return (ArgumentListSyntax)this.EatNode();
             }
 
-            SyntaxToken openToken, closeToken;
-            SeparatedSyntaxList<ArgumentSyntax> arguments;
-            ParseArgumentList(out openToken, out arguments, out closeToken, SyntaxKind.OpenParenToken, SyntaxKind.CloseParenToken);
-
+            ParseArgumentList(out var openToken, out var arguments, out var closeToken, SyntaxKind.OpenParenToken, SyntaxKind.CloseParenToken);
```

Comment:
>var [](start = 34, length = 3)

Please spell out types.

Reply:
Recorded in https://github.com/dotnet/roslyn/issues/23222